packages/sources/svelte/rollup.config.js

```diff
@@ -22,6 +22,10 @@ const src = {
       folder: 'navigation',
       components: ['VtmnSearch'],
     },
+    {
+      folder: 'navigation',
+      components: ['VtmnSearch', 'VtmnNavbar'],
+    },
```

Comment:
We always have this line just above (line 23), you don't need to add this new object.